ktlint/src/main/kotlin/com/pinterest/ktlint/Main.kt

```diff
@@ -784,6 +799,29 @@ object Main {
                 map
             }
 
+    /**
+     * Parses the .ktlintignore file, a list of ids of rules to globally disable. One rule per line.
+     */
+    private fun parseKtlintIgnore(path: String) {
```

Comment:
I think, this function should repeat the logic of `.editorconfig` lookup - https://editorconfig.org/#file-location

Implementation: https://github.com/pinterest/ktlint/blob/6f8bb56709e5394497a372402c7570326ffc32dd/ktlint/src/main/kotlin/com/pinterest/ktlint/internal/EditorConfig.kt#L20

This would allow to define global ktlint rules in project root and have sub-project specific one, that will enhance/override global rules.